packages/tasks/src/model-libraries.ts

```diff
@@ -275,6 +275,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	"mars5-tts": {
+		prettyLabel: "MARS5-TTS",
+		repoName: "MARS5-TTS",
+		repoUrl: "https://github.com/Camb-ai/MARS5-TTS",
+		filter: false,
+		countDownloads: { wildcard: { path: "*nar.safetensors" } },
```

Comment:
Why not `*ar.safetensors` then?

Reply:
See also: https://github.com/huggingface/huggingface.js/pull/749

(I think this won't work after that is merged)

Reply:
But it could be something like this, if the filenames are consistent:

`path_filename:"mars5_nar.safetensors" OR path_filename:"mars5_ar.safetensors"`

Reply:
Makes sense and good call!

Reply:
Fixed it!